src/js/components/landing/recentActivity/SubmissionStatus.jsx

```diff
@@ -63,11 +63,15 @@ export class SubmissionStatus extends React.Component {
 	render() {
 
 		const colors = this.progressBar(this.props.status);
+		let label = this.statusStrings[this.props.status]
```

Comment:
Double Checked with Kaitlin. Since they can have different status besides update (validating, validated, validated with errors, etc.) it was better to just add (need recert) after the fact